USAGE.rst

```diff
@@ -93,19 +95,21 @@ Java (pubsub)
   ::
 
      python execute_pipeline.py \
-        --config "../googleapis/gapic/core/artman_core.yaml:core,\
-        ../googleapis/gapic/lang/common.yaml:default|java" \
+        --config "../googleapis/gapic/core/artman_core.yaml,\
+        ../googleapis/gapic/lang/common.yaml" \
         JavaCorePipeline
```

Comment:
It would be handy if you just had to say "CorePipeline" on the command line for all languages, and have the --language argument find the right implementing class (JavaCorePipeline for Java)